db/migrate/20200203145050_create_versions.rb

```diff
@@ -0,0 +1,23 @@
+# This migration creates the `versions` table, the only schema PT requires.
+# All other migrations PT provides are optional.
+class CreateVersions < ActiveRecord::Migration[5.2]
+
+  # The largest text column available in all supported RDBMS is
+  # 1024^3 - 1 bytes, roughly one gibibyte.  We specify a size
+  # so that MySQL will use `longtext` instead of `text`.  Otherwise,
+  # when serializing very large objects, `text` might not be big enough.
+  TEXT_BYTES = 1_073_741_823
+
+  def change
+    create_table :versions do |t|
+      t.string   :item_type, {:null=>false}
+      t.integer  :item_id,   null: false, limit: 8
```

Comment:
Should ` item_id` be a UUID rather than an integer? I think all our models use UUIDs

Reply:
Yes - now I'm curious how any of the tests pass

Reply:
good spot - it just happens to work by accident.